WordPress/Classes/ViewRelated/Themes/ThemeBrowserCell.swift

```diff
@@ -209,7 +209,7 @@ open class ThemeBrowserCell: UICollectionViewCell {
         var queryItems: [URLQueryItem] = components.queryItems ??  []
 
         if let screenshotWidth = presenter?.screenshotWidth {
-            queryItems.append(URLQueryItem(name: "w", value: "\(presenter!.screenshotWidth)"))
+            queryItems.append(URLQueryItem(name: "w", value: "\(screenshotWidth)"))
```

Comment:
Did swift notice that `presenter` can't be `nil` here? If so, that's super cool!

Reply:
Nope 😢 

![image](https://user-images.githubusercontent.com/1218433/134425241-e77ab137-b137-44f2-93fb-4200a607a939.png)

Reply:
It took me a while to understand the question.  The reason I noticed we could just use `screenshotWidth` directly is because of the alert that @mokagio shows in his screenshot.